src/Components/StyledPostItem.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const CardItem = styled.div`
```

Comment:
You can add all these CSS properties to your CardArticle (as a wrapper for your component) and then you can delete this div.